### PR TITLE
[polaris.shopify.com] Tweak design of icons PR

### PR DESCRIPTION
**Pull request by martenbjork**

### CODE REVIEW

File: polaris.shopify.com/src/components/IconDetails/IconDetails.tsx [hunk 1]
```diff
@@ -15,131 +14,128 @@ interface Props {
...
-          <p>Select an icon</p>
-        </div>
-      </div>
-    );
-  }
+  if (!fileName) return <EmptyState />;
 
   const { set, description, name, keywords } = iconData;
+
+  const reactExamples = {
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/IconDetails/IconDetails.tsx:21]: I noticed that I accidentally messed up the indentation of these examples when I was working in this file. It's easy to indent a section and not realize that the examples get indented too.

Inlining this code looks a bit ugly, but helps prevent accidental indentation changes.

File: polaris.shopify.com/src/components/IconDetails/IconDetails.tsx [hunk 2]
```diff
@@ -15,131 +14,128 @@ interface Props {
...
+    imports: `import {\n  ${fileName}\n} from '@shopify/polaris-icons';`,
+    componentUsage: `<Icon\n  source={${fileName}}\n  color="base"\n/>`,
```
Comment (martenbjork) [inline polaris.shopify.com/src/components/IconDetails/IconDetails.tsx:23]: I noticed that I accidentally messed up the indentation of these examples when I was working in this file. It's easy to indent a section and not realize that the examples get indented too.

Inlining this code looks a bit ugly, but helps prevent accidental indentation changes.

**Review by alex-page — APPROVED** [review]
This is ace. Thank you @martenbjork.

File: polaris.shopify.com/src/components/SearchField/SearchField.tsx [hunk 3]
```diff
@@ -7,14 +9,18 @@ interface Props extends Omit<HTMLProps<HTMLInputElement>, "onChange"> {
...
-      type="search"
-      className={styles.SearchField}
-      onChange={(evt) => {
-        onChange && onChange(evt.target.value);
-      }}
-      {...props}
-    />
+    <div className={styles.SearchField}>
+      <div className={styles.Icon}>
+        <Image src={filterMajor} width={20} height={20} alt="" icon />
```
Comment (alex-page) [inline polaris.shopify.com/src/components/SearchField/SearchField.tsx:14]: We can probably replace this with `/icons/whateversearchiscalled.svg` but this is super.

**Pull request merged**